tests/data/ext/sample_ext_with_limits-1.4.0/HandlerConfiguration.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
I don't see the difference between this configuration and the malformed one?

Reply:
Do you mean the one in the test `test_handler_configuration_incorrect_memory_configuration` - I added `"memory_oom_kill": "enabledd"` <- the op is incorrect.

And for the test is `test_handler_configuration_incorrect_configuration` - It is trying to replicate different types of specs for different cases - Empty resource, no configs, etc.

Reply:
That's the difference, I didn't notice the typo, thanks.